modules/meeting/app/components/meetings/header_component.html.erb

```diff
@@ -1,6 +1,15 @@
 <%=
   component_wrapper do
-    render(Primer::OpenProject::PageHeader.new(state: @state)) do |header|
+    render(Primer::OpenProject::PageHeader.new(
+      test_selector: "meeting-page-header",
+      state: @state,
+      data: {
+        controller: "poll-for-changes",
+        poll_for_changes_reference_value: @meeting.changed_hash,
+        poll_for_changes_url_value: check_for_updates_meeting_path(@meeting),
+        poll_for_changes_interval_value: check_for_updates_interval,
+      }
```

Comment:
Not adding this data unless `OpenProject::FeatureDecisions.meeting_updated_notification_active?` will really include less of the feature in running code